src/main/java/me/totalfreedom/totalfreedommod/AntiSpam.java

```diff
@@ -114,4 +133,22 @@ public void onPlayerCommandPreprocess(PlayerCommandPreprocessEvent event)
             event.setCancelled(true);
         }
     }
-}
\ No newline at end of file
+
+    @EventHandler(priority = EventPriority.NORMAL)
+    public void onPlayerKick(PlayerKickEvent event)
+    {
+        if (markedfordeath.contains(event.getPlayer().getName()))
+        {
+            markedfordeath.remove(event.getPlayer().getName());
+        }
+    }
+    
+    @EventHandler(priority = EventPriority.NORMAL)
+    public void onPlayerLogin(PlayerLoginEvent event)
+    {
+        if (event.getPlayer().getName().startsWith("McStorm_"))
```

Comment:
Not ideal

Reply:
ye

Reply:
added that cuz there were bots with "mcstorm" as a prefix, you can remove it if you want idrc